FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Extensions/ConfigurationRegistrationExtensions.cs

```diff
@@ -78,9 +89,64 @@ private static void ValidateConfiguration(this IServiceCollection services)
                 .GetRequiredService<IOptions<SchemaConfiguration>>()
                 .Value;
 
-            if (schemaConfiguration.EnableCustomizedSchema && string.IsNullOrEmpty(schemaConfiguration.CustomizedSchemaImageReference))
+            if (string.IsNullOrWhiteSpace(schemaConfiguration.SchemaImageReference))
+            {
+                if (schemaConfiguration.EnableCustomizedSchema)
+                {
```

Comment:
Do we need validate if the `SchemaCollectionDirectory` exists here?

Reply:
Removed ```SchemaCollectionDirectory```